action.yml

```diff
@@ -0,0 +1,26 @@
+name: Git Town - GitHub Action
+description: Visualizes your stacked changes when proposing pull requests on GitHub
+
+branding:
+  icon: 'home'
+  color: 'orange'
+
+inputs:
+  github-token:
+    required: true
+    default: ${{ github.token }}
+  main-branch:
+    required: false
+    default: ''
+  perennial-branches:
+    required: false
+    default: ''
+  template:
+    required: false
+    default: |
+      ## Stack
+      {{ stack }}
```

Comment:
Is this still used? I don't see it referenced anywhere.

Reply:
No! Good catch. I meant to remove the `template` input but removed `perennial-regex` instead. Don't code at 1am in the morning!